### ISSUE TITLE
[Bug]: @tiptap/extension-text-style is suddenly required in 2.8.0

### ISSUE THREAD

**Issue by Nantris** [ueberdosis/tiptap#5707]
### Affected Packages

extension-bullet-list

### Version(s)

2.8.0

### Bug Description

Uncaught Error: Cannot find module '@tiptap/extension-text-style'

### Browser Used

Chrome

### Expected Behavior

No new dependencies are needed, or if they are they are automatically managed.

### Dependency Updates

- [X] Yes, I've updated all my dependencies.

**Comment by yroc92** [comment]
Also seeing this. Here's our error (nextjs):

```
Module not found: Can't resolve '@tiptap/extension-text-style'

https://nextjs.org/docs/messages/module-not-found

Import trace for requested module:
../node_modules/@tiptap/starter-kit/dist/index.js
```

**Comment by bbaappss** [comment]
Yup, same..

Had to do this 

<img width="584" alt="image" src="https://github.com/user-attachments/assets/81673761-cc11-4b67-9067-9aa6d8069e1f">

But that only works on local, not sure if I can get this to my production..

**Comment by guanriyue** [comment]
The following diff is the relevant changes.

https://github.com/ueberdosis/tiptap/commit/6834a7f7d620c1706aaa5220db3ffcfebef946b7#diff-bc4c95e6db01b15dbca4d9271aec52b402faf0c9a0d5a1d02aaf14cf87056a60

The way packages were referenced in the past code was "wrong". `@tiptap/extension-ordered-list` has the same problem

The code changes from `import TextStyle from '../../extension-text-style/src/index.js'` to `import { TextStyle } from '@tiptap/extension-text-style'`.

In this change, `@tiptap/extension-text-style` is added as peerDeps, which means you have to add the dependency manually.

This looks like a breaking change.

**Comment by Nantris** [comment]
So I guess it should just be a dependency instead of a peerDependency, yeah?

**Comment by guanriyue** [comment]
Yes, you are right.

**Comment by nperez0111** [comment]
This is released with v2.9.0
